libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/pivotTable/tests/modifyInsightForDrillDownMock.ts

```diff
@@ -0,0 +1,222 @@
+// (C) 2020 GoodData Corporation
+
+import {
+    bucketSetTotals,
+    IFilter,
+    IInsight,
+    IInsightDefinition,
+    ISortItem,
+    modifyAttribute,
+    newAttribute,
+    newAttributeSort,
+    newBucket,
+    newInsightDefinition,
+    newPositiveAttributeFilter,
+    newTotal,
+    uriRef,
+} from "@gooddata/sdk-model";
+import { IImplicitDrillDown, IVisualizationProperties } from "../../../..";
+import { Department, Region, Status, Won } from "@gooddata/reference-workspace/dist/ldm/full";
+
+const properties: IVisualizationProperties = {
+    controls: {
+        columnWidths: [
+            {
+                attributeColumnWidthItem: {
+                    attributeIdentifier: Region.attribute.localIdentifier,
+                    width: { value: 131 },
+                },
+            },
+            {
+                attributeColumnWidthItem: {
+                    attributeIdentifier: Status.attribute.localIdentifier,
+                    width: { value: 125 },
+                },
+            },
+            {
+                attributeColumnWidthItem: {
+                    attributeIdentifier: Department.attribute.localIdentifier,
+                    width: { value: 97 },
+                },
+            },
+            { measureColumnWidthItem: { width: { value: 270 } } },
+        ],
+    },
+    sortItems: [
+        {
+            attributeSortItem: {
+                attributeIdentifier: Region.attribute.localIdentifier,
+                direction: "desc",
+            },
+        },
+        {
+            attributeSortItem: {
+                attributeIdentifier: Status.attribute.localIdentifier,
+                direction: "asc",
+            },
+        },
```

Comment:
please use factory functions to create all this. pivot table width items have factories in sdk-ui-pivot, sort items have factories in sdk-model

Reply:
Done